### DISCUSSION TITLE
Endless dependencies

### DISCUSSION THREAD

**Discussion by StephanieBeals** [pschatzmann/ESP32-A2DP#537] [Q&A] [answered]
Hello! 
I downloaded the library using git. 

I am getting a lot of issues every time I try to run the code I get a dependancy for a library that I don't have. Is that normal or did I do something wrong?

**Comment by pschatzmann** [comment, marked as answer]
What are you trying to run and what error messages are you getting ?

I think I forgot to merge some open changes. Try to update using: git pull

ps. don't forget that the example from the latest version require the [AudioTools](https://github.com/pschatzmann/arduino-audio-tools) library to be installed, [unless you continue to use the legacy I2S API](https://github.com/pschatzmann/ESP32-A2DP/wiki/Legacy-I2S-API)
